src/hotspot/share/classfile/verifier.cpp

```diff
@@ -290,13 +291,12 @@ bool Verifier::is_eligible_for_verification(InstanceKlass* klass, bool should_ve
     !(klass->is_shared() && klass->is_rewritten()) &&
 
     // As of the fix for 4486457 we disable verification for all of the
-    // dynamically-generated bytecodes associated with the 1.4
-    // reflection implementation, not just those associated with
+    // dynamically-generated bytecodes associated with
     // jdk/internal/reflect/SerializationConstructorAccessor.
     // NOTE: this is called too early in the bootstrapping process to be
     // guarded by Universe::is_gte_jdk14x_version().
     // Also for lambda generated code, gte jdk8
```

Comment:
While you are here could you delete these version comments please - they are meaningless these days. Thanks.

Reply:
Done.